nova/conf/vmware.py

```diff
@@ -114,35 +114,38 @@
 For big VMs as determined by the `bigvm_mb` setting, this setting is not used.
 Big VMs always reserve all their memory.
 """),
-    cfg.IntOpt('memory_reservation_cluster_hosts_max_fail',
+    cfg.IntOpt('resource_reservation_cluster_hosts_max_fail',
                 default=0,
                 min=0,
                 help="""
-Allow reserving instance memory while at least n hypervisors of memory remain
-unreserved in the cluster. This is a safety margin so a certain number of
-cluster hypervisors are allowed to fail. If more memory would be reserved and
-all the anticipated HV failures occurred, existing VMs with reserved memory
-would no longer be able to start.
+Allow reserving instance resources while at least n hypervisors of the resource
+remain unreserved in the cluster. This is a safety margin so a certain number
+of cluster hypervisors are allowed to fail. If more resources would be reserved
+and all the anticipated HV failures occurred, existing VMs with reserved
+resources would no longer be able to start.
 
 This setting applies to VMs with flavors which have a nonzero extra_spec
-"resources:CUSTOM_MEMORY_RESERVABLE_MB" set.
+"resources:CUSTOM_CPU_RESERVABLE" or "resources:CUSTOM_MEMORY_RESERVABLE_MB"
+set.
```

Comment:
This sentence makes this setting sound like it's exclusively for VMs with these extra_specs. But that's not true, right? We also have a setting stating the memory over which every VM is related to this setting, too.

In general, it doesn't sound like this setting has anything to do with VMs but rather with the driver reporting resources. Not sure why we mention them here at all.

Reply:
removed the paragraph in both configs.